shell/platform/embedder/embedder_surface_gl.cc

```diff
@@ -11,18 +11,26 @@ namespace flutter {
 EmbedderSurfaceGL::EmbedderSurfaceGL(
     GLDispatchTable gl_dispatch_table,
     bool fbo_reset_after_present,
+<<<<<<< HEAD
```

Comment:
Bad merge.

Reply:
Yeah, forgot to save the file haha. Fixing.